src/js/components/TodoInput.js

```diff
@@ -0,0 +1,30 @@
+import { $ } from "../lib/util.js";
+import { ERROR_MESSAGES } from "../constants/message.js";
+import { MINIMUM_LENGTH } from "../constants/limitValue.js";
+import { KEY } from "../constants/eventKey.js";
+
+class TodoInput {
+  constructor({ onAddItem }) {
+    this.onAddItem = onAddItem;
+    this.init();
+  }
+
+  init() {
+    this.registerEventListener();
+  }
+
+  registerEventListener() {
+    $(".new-todo").addEventListener("keydown", (event) => this.addTodoItem(event));
+  }
+
+  addTodoItem = (event) => {
+    if (event.key !== KEY.ENTER) return;
+    if (event.target.value.length < MINIMUM_LENGTH.ITEM_CONTENTS)
+      return alert(ERROR_MESSAGES.TOO_SHORT_ITEM_CONTENTS);
```

Comment:
```suggestion
    if (event.target.value.length < MINIMUM_LENGTH.ITEM_CONTENTS) {
      return alert(ERROR_MESSAGES.TOO_SHORT_ITEM_CONTENTS);
    }
```
if문이 길어지면 brace를 넣으시는 건 어떨까요?
[Airbnb 스타일 가이드](https://github.com/airbnb/javascript#blocks--braces) 에서도 그렇게 제안하더라구요.